source/basic.tex

```diff
@@ -5168,12 +5167,16 @@
 \end{example}
 
 \pnum
+\indextext{array!\idxcode{const}}%
+An array type whose elements are cv-qualified
+is also considered to have the same cv-qualifications
+as its elements.
+\begin{note}
 Cv-qualifiers applied to an array
 type attach to the underlying element type, so the notation
```

Comment:
We need two rules:

 1) `identity_t<const T>[N]` is a const-qualified type because its element type is const-qualified.
 2) `const identity_t<int[3]>` has a const-qualified element type.

Before this change, we say (2) in two places (unless I missed one). This change removes one of them (on line 5126) and converts the other (here) to a note, removing any normative specification of this rule. The new normative text (lines 5170-5174) says only (1), so we're left with nowhere that provides rule (2).